### PR TITLE
Tooltips UI 

### PR DESCRIPTION
**Pull request by pkattera**
# Description

Currently it can be hard to distinguish between tooltips and the background content, since there is not much contrast behind the tooltip. This PR:
- Adds a deeper shadow to light mode
- Adds a colorful radial gradient to dark mode.
- Adds a bottom and top triangle towards the tooltip trigger
- Adds a small upwards / downwards animation

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [x] Other: UI improvement

## How should this be tested?

View as many tooltips as possible to ensure none of them have broken and that the padding looks good.

## Visual context

Screen Shot 2022-04-20 at 4 17 52 PM

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have requested at least 2 reviews (If the PR is significant enough, use best judgement here)
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [ ] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/components/_global/BalTooltip/BalTooltip.vue [hunk 1]
```diff
@@ -83,22 +88,105 @@ onMounted(() => {
...
+.tooltip[data-popper-placement='bottom'] .tooltip-content {
+  opacity: 0;
+  animation: fadeInMoveDown 0.2s ease-out both;
+}
+
+/* bottom triangle for top placed tooltips */
+.tooltip[data-popper-placement='top'] .tooltip-content:before {
+  border-top: solid #fff 8px;
+  border-left: solid transparent 8px;
+  border-right: solid transparent 8px;
```
Comment (garethfuller) [inline src/components/_global/BalTooltip/BalTooltip.vue:155]: I'm a little concerned with all the custom sizing and hex colors here and below. Is it difficult to use to tailwind classes here?

**Review by garethfuller — APPROVED** [review]
Looks great! Just slightly concerned with all the custom CSS, if we can use tailwind as much as possible throughout that, that would be great.

**Comment by pkattera** [comment]
> Looks great! Just slightly concerned with all the custom CSS, if we can use tailwind as much as possible throughout that, that would be great.

I've removed all custom CSS that can be replaced by Tailwind v2 classes

File: src/components/_global/BalTooltip/BalTooltip.vue (hunk 1 above)
Comment (garethfuller) [reply src/components/_global/BalTooltip/BalTooltip.vue:156]: Ahh ok, I didn't realise that was not possible in v2 tbh.

**Review by garethfuller — APPROVED** [review]

**Pull request merged**